pkgs/by-name/oa/oatmeal/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+system ? builtins.currentSystem
+, pkgs
```

Comment:
Do not use `pkgs` as a parameters. As `callPackage` is already being used to call this, just pick the dependencies you need.